src/server/pps/cmds/cmds.go

```diff
@@ -438,6 +439,24 @@ each datum.`,
 		follow      bool
 		tail        int64
 	)
+
+	// prettyLogsPrinter helps to print the logs recieved in different colours
+	prettyLogsPrinter := func(message string) {
+		informationArray := strings.Split(message, " ")
+		debugLevel := strings.ToLower(informationArray[1])
+		if debugLevel == "info" {
+			color.Set(color.FgGreen)
+		} else if debugLevel == "warning" {
+			color.Set(color.FgYellow)
+		} else if debugLevel == "error" {
+			color.Set(color.FgRed)
+		} else {
+			color.Set(color.FgWhite)
+		}
+		fmt.Println(message)
```

Comment:
I wonder if we should print the full line in color (as is the case here), or just the log level in color -- @jdoliner any preferences?